rasa/shared/nlu/training_data/training_data.py

```diff
@@ -767,6 +774,16 @@ def has_e2e_examples(self):
         return any(message.is_e2e_message() for message in self.training_examples)
 
 
+RELEVANT_MESSAGE_KEYS = [
```

Comment:
We most likely need to add ACTION_TEXT and ACTION_NAME for e2e training.

Reply:
@dakshvar22 do you remember why we need that list of constants?

Reply:
I think adding `ACTION_TEXT, ACTION_NAME` is enough. I think we need `TOKEN_NAMES[TEXT]` for entities